app/uk/gov/hmrc/nisp/views/account_forecastonly.scala.html

```diff
@@ -0,0 +1,68 @@
+@*
+* Copyright 2015 HM Revenue & Customs
```

Comment:
@dspork Looking at other open source projects we should have all years since creation. For example "2015, 2016"